src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+
+<figure class="w-figure">
+  <img src="./image14.png" alt="The Podcasts app with the 'Episode Notes' drawer expanded." width="600">
+  <figcaption class="w-figcaption">In-app animations like when opening a drawer are snappy.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  Performant animations on the web are certainly possible if you take into account a number of best practices
+  outlined in the article <a href="https://developers.google.com/web/fundamentals/design-and-ux/animations/animations-and-performance">Animations and Performance</a>.
+  Scroll animations as commonly seen in paginated content or media carousels can be massively improved by using the <a href="https://developers.google.com/web/updates/2018/07/css-scroll-snap">CSS Scroll Snap</a> feature.
+  For full control, you can use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API">Web Animations API</a>.
+{% endDetails %}
+
+## Content surfaced outside of app
+
+The Podcasts app on iOS can surface content in other locations than the actual application, for example, in the system's Widgets view, or in the form of a Siri Suggestion. Having proactive, usage-based calls-to-action that just require a tap to interact with can greatly increase the re-engagement rate of an app like Podcasts.
+
+<figure class="w-figure">
+  <img src="./image6.png" alt="iOS Widget view showing the Podcasts app suggesting a new episode of a podcast." width="300">
+  <figcaption class="w-figcaption">App content is surfaced outside of the main Podcasts app.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/content-indexing-api/">Content Index API</a> allows your application
+  to tell the browser which content of the PWA is available offline.
+  This allows the browser to surface this content outside of the main app.
+  By marking up interesting content in your app as suitable for <a href="https://developers.google.com/search/docs/data-types/speakable">speakable</a>
+  audio playback and by using <a href="https://developers.google.com/search/docs/guides/search-gallery">structured markup</a> in general,
+  you can help search engines and virtual assistants like the Google Assistant present your offerings in an ideal light.
+{% endDetails %}
+
+## Lock screen media control widget
+
+When a podcast episode is playing, the Podcasts app shows a beautiful control widget on the lock screen that features metadata like the episode artwork, the episode title, and the podcast name.
+
+<figure class="w-figure">
+  <img src="./image17.png" alt="iOS media playback widget on the lock screen showing a podcast episode with rich metadata." width="300">
```

Comment:
```suggestion
  <img src="./image17.png" class="w-screenshot w-screenshot--filled" alt="iOS media playback widget on the lock screen showing a podcast episode with rich metadata." width="300">
```

Reply:
This uses a device frame now, no screenshot class needed.